yarn.lock

```diff
@@ -1730,19 +1729,6 @@
     "@material-ui/utils" "^4.0.0-alpha.8"
     "@types/react-transition-group" "^2.0.16"
     clsx "^1.0.2"
-=======
-  version "3.9.3"
-  resolved "https://registry.yarnpkg.com/@material-ui/core/-/core-3.9.3.tgz#d378c1f4beb18df9a534ca7258c2c33fb8e0e51f"
-  integrity sha512-REIj62+zEvTgI/C//YL4fZxrCVIySygmpZglsu/Nl5jPqy3CDjZv1F9ubBYorHqmRgeVPh64EghMMWqk4egmfg==
-  dependencies:
-    "@babel/runtime" "^7.2.0"
-    "@material-ui/system" "^3.0.0-alpha.0"
-    "@material-ui/utils" "^3.0.0-alpha.2"
-    "@types/jss" "^9.5.6"
-    "@types/react-transition-group" "^2.0.8"
-    brcast "^3.0.1"
-    classnames "^2.2.5"
->>>>>>> css-utils README with JSDoc generated API
```

Comment:
Looks like you've accidentally committed unresolved merge conflicts.

Reply:
indeed, the commit right after fixed this, almost sure it's a bug in Atom.

Reply:
Ah, okay, sorry. The GitHub notification linked me to this commit. I didn't spot the follow-up.